spec/components/location_code_facet_component_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe LocationCodeFacetComponent, type: :component do
+  before { stub_holding_locations }
+
+  describe '#location_codes_by_lib' do
+    let(:location_items) { [architecture, rcppw, ues, uesla, uesrf] }
+    let(:architecture) { Blacklight::Solr::Response::Facets::FacetItem.new(hits: '792', value: 'Architecture Library') }
+    let(:rcppw) { Blacklight::Solr::Response::Facets::FacetItem.new(hits: '792', value: 'arch$pw') }
+    let(:ues) { Blacklight::Solr::Response::Facets::FacetItem.new(hits: '792', value: 'arch$circ') }
+    let(:uesla) { Blacklight::Solr::Response::Facets::FacetItem.new(hits: '792', value: 'arch$la') }
+    let(:uesrf) { Blacklight::Solr::Response::Facets::FacetItem.new(hits: '792', value: 'arch$ref') }
+
+    subject(:codes) do
+      component = described_class.new(display_facet: nil, label: 'label', blacklight_config: nil, search_state: nil)
+      component.location_codes_by_lib(location_items)
+    end
+
+    it 'includes library name as key' do
+      expect(codes.key?('Architecture Library')).to be true
+    end
+    describe 'architecture library' do
+      let(:architecture_hash) { subject['Architecture Library'] }
+
+      it 'includes holding location code facet items' do
+        # In Voyager the `architecture_hash['codes']` did not include the ReCAP library (rcppw)
```

Comment:
Can we remove this comment? I understand it's from an existing spec but it's incorrect. 
It would be nice to update the architecture codes to reflect the correct ones. By updating the stub this spec will fail with the existing incorrect codes. See the architecture codes in [bibdata/holding_locations](https://bibdata.princeton.edu/locations/holding_locations) or some of them in [stub alma holdings](https://github.com/pulibrary/orangelight/blob/dc2b4d4ab08432c98be38100fd60a822d7cb51b5/spec/fixtures/bibdata/alma/holding_locations.json#L364).